gnat2goto/driver/ureal_to_binary.adb

```diff
@@ -71,17 +71,11 @@ package body Ureal_To_Binary is
          --  writing to the first position
          Int_End := Int_Part'First + Integer (E);
 
-         if Int_End > Int_Part'Last then
-            raise Integer_Part_Too_Large;
-         end if;
-
-         pragma Assert (Int_End <= Int_Part'Last);
          --  use the definition of the binary representation to get the integer
          --  part of the fraction, i.e. our integer part is going to be b_E
          --  b_(E-1) ... b_0 (binary digits) with the integer part of
          --  our number being sum (n <- 0..E) { b_n * 2^n}
-         while E >= Uint_0 loop
-            pragma Assert (Digit_Position <= Int_End);
+         while E >= Uint_0 and then Digit_Position <= Int_End loop
```

Comment:
Same here

Reply:
The check for `digit_position <= int_end` is redundant but it might be nice to keep as documentation.

Reply:
No, I mean I don't think you need this change, this assertion should always pass now